src/mcp/shared/auth.py

```diff
@@ -117,17 +117,17 @@ class OAuthMetadata(BaseModel):
     response_modes_supported: list[Literal["query", "fragment", "form_post"]] | None = None
     grant_types_supported: list[str] | None = None
     token_endpoint_auth_methods_supported: list[str] | None = None
-    token_endpoint_auth_signing_alg_values_supported: None = None
+    token_endpoint_auth_signing_alg_values_supported: list[Literal["RS256", "ES256"]] | None = None
```

Comment:
With my previous comment I didn't mean to set the `Literal`.

I think the above also had `Literal` before but we had to switch to `str` because of something I can't remember... I would have to check.

Can you change it to `list[str]`?

---

Happy to approve after it.